tachyon/math/polynomials/univariate/univariate_evaluation_domain.h

```diff
@@ -235,6 +235,13 @@ class UnivariateEvaluationDomain : public EvaluationDomain<F, MaxDegree> {
     return F::GetSuccessivePowers(size, root);
   }
 
+  // Computes the first |size| roots of unity for the entire domain.
+  // e.g. for the domain [1, g, g², ..., gⁿ⁻¹}] and |size| = n / 2, it
+  // computes [1, g, g², ..., g^{(n / 2) - 1}]
+  constexpr std::pmr::vector<F> GetRootsOfUnityPmr(size_t size, const F& root) const {
+    return F::GetSuccessivePowersPmr(size, root);
+  }
```

Comment:
What about keeping only one `GetRootsOfUnity()` that uses `std::pmr::vector`? If we have two versions, users might get confused. Additionally, this typically generates long series of powers, so having a single, efficient implementation would be beneficial.

Reply:
It is used in [`radix2_evaluation_domain`](https://github.com/kroma-network/tachyon/blob/e8558ecddb6b36dfc0d63e78cf5e8c940844eb06/tachyon/math/polynomials/univariate/radix2_evaluation_domain.h#L300). So the change can propagate to packed prime fields as well.